x-pack/plugins/security_solution/public/common/components/tables/helpers.tsx

```diff
@@ -97,23 +106,30 @@ export const RowItemOverflowComponent: React.FC<RowItemOverflowProps> = ({
             <MoreContainer
               fieldName={fieldName}
               idPrefix={idPrefix}
-              values={values}
+              values={values.slice(0, maxOverflowItems + 1)}
```

Comment:
👍🏾 , just move this out of the `return ()` as `.slice()` is going to create a new array on every re-render causing all the children of `MoreContainer` to re-render. May not be a big deal since it's a popover, but just good practice. 

```
const  maxVisibleValues = useMemo(() => values.slice(0, maxOverflowItems + 1), [values, maxOverflowItems]);
....

<MoreContainer values={maxVisibleValues} />

```

Reply:
Good point! Thanks - will change that 😄